### ISSUE TITLE
Add Option for Mobile Code Verification Instead of Challenge Question for CROMMER Signing

### ISSUE THREAD

**Issue by MaeganWood64** [US-EPA-CAMD/PI-1#4568]
- Currently the only option for verification for CROMMER Signing is a Challenge Question. An additional option for Mobile Code Verification is to be added 

**User Story** 
As an ECMPS User in the Submit Process
I want the option to be able to have a mobile code sent to me
So that I am able to verify myself and complete the signature process 

**Requirements** 
Given a User is moving through the Submit Modal window
When the Modal Appears 
And the User has verified their Credentials 
Then an option will appear to verify themselves using a Challenge Question or a Mobile Code

Given a User is on the Verification Section of the Modal Window
When they select the radio option for "Send Text Message" 
Then a field will appear with the following objects: 
- Heading: "Text Message will be sent to:'
- Dropdown field with the users phone information from their CDX account
- Send button
- "Text or Data rates may apply" 

Given a user has selected the "Send Text Message Option" 
When they press send
Then a text message will be sent to their listed phone number with a code
and the "Send" button text will be changed to "Resend" 

Given a user has selected the "Send Text Message Option" 
and they have recieved the code from CDX Soap Services
When they enter the PIN and press verify
and the Pin is Correct 
then they are moved to the Certification Statement Section of the Modal Window

Given a user has selected the "Send Text Message Option" 
and they have recieved the code from CDX Soap Services
When they enter the PIN and press verify
and the Pin is incorrect
then an error message appears reading "Insert from CDX Mobile Verification Process" 

**Dev Notes**
- Use integrations with CDX to implement Mobile Code Functionality 
- Design to align with current CDX Verfication options![image (8).png](https://images.zenhubusercontent.com/60e852f1b32c48306fa0dd84/0c7f3a5b-6d8e-4dfb-8142-703487a6acd6)

-[Wireframes for Review& Submit](https://projects.invisionapp.com/freehand/document/oboCJmAfA)

**Testing**
Test Case: 
Tester: Maegan Wood

**Comment by MaeganWood64** [comment]
- Need PD Session for this one

**Comment by MaeganWood64** [comment]
- waiting on confirmation from CDX higherups, but number param is passed in with cdx user information and mobile number used for verification is the one used in CDX signup

**Comment by JanellC** [comment]
Will not be addressed in PI 13. There are discussion of removing the verfication all together. Will review later on

**Comment by JanellC** [comment]
Ticket no longer needed. Mobile verficiation will be removed from review and submit
